unicore/ask/service/schema.py

```diff
@@ -30,3 +30,36 @@ class QuestionSchema(colander.MappingSchema):
         validator=validators.question_type_validator)
     multiple = colander.SchemaNode(colander.Boolean(), default=False)
     options = Options(validator=validators.options_validator, default=[])
+
+
+class QuestionOptionSchemaPut(colander.MappingSchema):
+    uuid = colander.SchemaNode(
+        colander.String(),
+        validator=validators.option_uuid_validator, missing=None)
+    title = colander.SchemaNode(
+        colander.String(),
+        validator=validators.option_title_validator, missing=None)
```

Comment:
Afaik `patch` is the partial update one. I'm fine with using `put` but the docs will have to be clear. Another issue then is that the endpoint does a partial update on the question, but a full update on the options